ASM/src/hacks.asm

```diff
@@ -1294,52 +1294,11 @@ nop
 ; Item menu
 ;==================================================================================================
 
-; Left movement check
-; Replaces:
-;   beq     s4, t5, 0x8038F2B4
-;   nop
-.orga 0xBB77B4 ; In memory: 0x8038F134
-    nop
-    nop
-
-; Right movement check
+; Reimplement KaleidoScope_DrawItemSelect
 ; Replaces:
-;   beq     s4, t4, 0x8038F2B4
-;   nop
-.orga 0xBB7894 ; In memory: 0x8038F214
-    nop
-    nop
-
-; Upward movement check
-; Replaces:
-;   beq     s4, t4, 0x8038F598
-;   nop
-.orga 0xBB7BA0 ; In memory: 0x8038F520
-    nop
-    nop
-
-; Downward movement check
-; Replaces:
-;   beq     s4, t4, 0x8038F598
-;   nop
-.orga 0xBB7BFC ; In memory: 0x8038F57C
-    nop
-    nop
-
-; Remove "to Equip" text if the cursor is on an empty slot
-; Replaces:
-;   addu    s1, t6, t7
-;   lbu     v0, 0x0000 (s1)
-.orga 0xBB7C88 ; In memory: 0x8038F608
-    jal     item_menu_prevent_empty_equip
-    addu    s1, t6, t7
-
-; Prevent empty slots from being equipped
-; Replaces:
-;   lbu     v0, 0x0000 (s1)
-;   addiu   at, r0, 0x0009
-.orga 0xBB7D10 ; In memory: 0x8038F690
-    jal     item_menu_prevent_empty_equip
+;   something I'll get back to
```

Comment:
What do you mean by this comment?

Reply:
If you look at all the other ASM hacks, people list which specific ASM instructions were replaced. It's generally in the format PJ64 uses, so I just have to open that address up in the debugger and copy what I see, but that means getting wine setup again.

Reply:
You are replacing this two instructions here

```mips
addiu       $sp, $sp, -0xA8
sw          $s5, 0x30($sp)
```

Reply:
Thank you.